.env.sample

```
@@ -9,7 +9,24 @@ DB_HOST=1.2.3.4
 DB_PORT=5432
 DB_USER=postgres
 DB_PASSWORD=postgres
-DB_NAME=tcs
+DB_NAME=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PASSWORD key

(UnorderedKey)